tools/shared.py

```diff
@@ -2957,6 +2973,13 @@ class JS(object):
 
   global_initializers_pattern = r'/\* global initializers \*/ __ATINIT__.push\((.+)\);'
 
+  emscripten_license = '''/**
```

Comment:
For multiline comments like this I find them easier to read if you start them with `''\`

Reply:
Oh, I didn't know about that, nice!